packages/api/test/perma-cache-post.spec.js

```diff
@@ -167,36 +164,43 @@ test('Fails when it was already perma cached', async (t) => {
   t.is(body, 'The provided URL was already perma cached')
 })
 
+test('Fails on concurrent perma cache attempt of same URL', async (t) => {
```

Comment:
I think this will be flakey unless you add some artificial delay.

Reply:
We would need to mock the IPFS Node instead of running a real one, which has been something we have been moving out of.

Either we just drop the test (it essentially validates a SQL `CREATE UNIQUE INDEX`) that should be well tested for now, or we keep it and see if it becomes flakey, removing if so. Atm, I have never seen it fail in multiple runs